code/Spikes/Class_WiFiService/Class_WiFiService.ino

```diff
@@ -0,0 +1,35 @@
+//M. Hamidi: This is a 'Main'-Like Ino File. Just to understand how to integrate the class in the 'real' main ino file in our repository.
+//Important: DON'T include SoftwareSerial.h it is already included by WiFiService.h
+
+#include <Arduino.h>
+#include "WiFiService.h"
+
+// Create WiFiService object
+WiFiService myWiFiService;
+
+void setup()
+{
+  Serial.begin(9600);     // Set data rate for the HW serial port
+  while (!Serial) {;}     // wait for HW serial port to connect.
+}
+
+
+
+void loop() {
+
+  myWiFiService.Init(true);
```

Comment:
Anfangs habe ich mich gefragt: "Muss wirklich in jedem loop der Service neu initialisiert werden?" Aber das "Init" steuert einfach den Empfang der Nachrichten?